firestore-palm-gen-text/extension.yaml

```diff
@@ -71,6 +71,10 @@ params:
     type: string
     required: false
     immutable: false
+    validationRegex: ^[A-Za-z]+(,[A-Za-z]+)*$
```

Comment:
```suggestion
    validationRegex: ^[a-zA-Z_][a-zA-Z0-9_]{0,99}(,[a-zA-Z_][a-zA-Z0-9_]{0,99})*$
```

Reply:
Fixes #39, Firestore allows numbers, underscores and capital letters in field names